modules/ossm-updating-istio-control-plane-with-revisionbased-istiorevisiontag.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+
+// update/ossm-updating-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-istio-control-plane-with-revisionbased-istiorevisiontag_{context}"]
+= Updating Istio control plane with RevisionBased strategy and IstioRevisionTag
+
+When updating {istio} using the `RevisionBased` strategy, you can create an `IstioRevisionTag` resource to tag a specific `IstioRevision` resource. You can use the `IstioRevisionTag` resource to attach workloads to a specific `IstioRevision` resource without modifying the `istio.io/rev` label on namespaces or pods.
+
+.Prerequisites
+
+* You are logged in to {ocp-product-title} as a user with the `cluster-admin` role.
+* You have installed the {SMProductName} Operator 3, and deployed {istio} with the `RevisionBased` strategy.
+* You have created an `IstioRevisionTag` resource and the `targetRef` field is referencing the desired `{istio}` resource.
+* You have installed the {istio} CNI plugin with the desired version.
+* You have application workloads running in the cluster.
+* You have labeled the bookinfo namespace to enable sidecar injection.
+* You have installed `istioctl` on your local machine.
+
+.Procedure
+
+. Change the version in the `{istio}` resource. For example, to update to {istio} `1.24.4`, set the `spec.version` field to `v1.24.4` by running the following command:
++
+[source,terminal]
+----
+$ oc patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.24.4"}}'
```

Comment:
Please review upstream [steps](https://github.com/istio-ecosystem/sail-operator/blob/main/docs/README.md#example-using-the-revisionbased-strategy-and-an-istiorevisiontag) (step 10. and further) and update this section.